client/packages.go

```diff
@@ -37,6 +39,9 @@ type Package struct {
 	Version       string `json:"version"`
 }
 
+// Packages are collections of Package items
```

Comment:
Although I very much appreciate comments, I'd like to discourage this specific kind of comment. You're probably not alone here, so I'm sorry for picking up on you specifically, but this was a good example to raise the point on. Comments should add something to the code text, otherwise they're just overload for the reader.